.github/workflows/generate_docs.yaml

```diff
@@ -0,0 +1,33 @@
+name: generate-docs
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "mkdocs.yaml"
+      - "**/*.md"
+      - "**/*.svg"
+      - "**/*.png"
+      - "**/*.css"
+  pull_request_target:
+    paths:
+      - "mkdocs.yaml"
+      - "**/*.md"
+      - "**/*.svg"
+      - "**/*.png"
+      - "**/*.css"
+
+jobs:
+  get-docs-version-name:
+    uses: autowarefoundation/autoware-github-actions/.github/workflows/reusable-get-docs-version-name.yaml@tier4/proposal
```

Comment:
We'll probably replace these workflows with `composite`. So please keep in mind that.

Reply:
I'd like to make issue for my reminder. Thanks!